server/devpi_server_pytest/__init__.py

```diff
@@ -0,0 +1,7 @@
+def pytest_addoption(parser):
+    parser.addoption(
+        "--slow", action="store_true", default=False,
+        help="run slow tests involving remote services (pypi.python.org)")
+    parser.addoption(
+        "--backend", action="store",
+        help="run tests with specified dotted name backend")
```

Comment:
hum, this introduces a "completion" conflict when i type "vim devpi_serve<TAB>" which currently does not exist.   You could also put the plugin into the test_devpi_server directory  which kind of makes sense i'd think.